daemon/algod/api/server/v2/handlers.go

```diff
@@ -1703,6 +1705,54 @@ func (v2 *Handlers) TealDisassemble(ctx echo.Context) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+// GetLedgerStateDeltaForTransactionGroup retrieves the delta for a specified transaction group.
+// (GET /v2/deltas/txn/group/{id})
+func (v2 *Handlers) GetLedgerStateDeltaForTransactionGroup(ctx echo.Context, id string, params model.GetLedgerStateDeltaForTransactionGroupParams) error {
+	handle, contentType, err := getCodecHandle((*string)(params.Format))
+	if err != nil {
+		return badRequest(ctx, err, errFailedParsingFormatOption, v2.Log)
+	}
+	idDigest, err := crypto.DigestFromString(id)
+	if err != nil {
+		return badRequest(ctx, err, errNoValidTxnSpecified, v2.Log)
+	}
+	tracer, ok := v2.Node.LedgerForAPI().GetTracer().(*eval.TxnGroupDeltaTracer)
+	if !ok {
+		return serviceUnavailable(ctx, err, errFailedRetrievingTracer, v2.Log)
+	}
```

Comment:
Does this handle a nil tracer?

Reply:
Yes, I've added a test for this case as well just to be sure.

I'm not sure that 503 is the exactly the correct http response though--open to suggestions on better responses there.

Reply:
I don't think 503 is a good choice, isn't it supposed to be fore temporary conditions? I'd probably go with 501 or 500.